src/login/LoginView.ts

```diff
@@ -216,7 +216,7 @@ export class LoginView extends BaseTopLevelView implements TopLevelView<LoginVie
 		return m(
 			".flex.col.pb",
 			{
-				oncreate: (vnode) => {
```

Comment:
it is not allowed to resolve the same promise twice. We need to recreate `this.loginForm` in `onDestroy()` of it I think.

Reply:
I agree that this approach is cleaner. I will ammend the changes since it is only a small change anyway